app/controllers/atmosphere/api/v1/appliance_sets_controller.rb

```diff
@@ -20,7 +20,13 @@ def show
         end
 
         def create
-          if conflicted? @appliance_set.appliance_set_type
+          if !@appliance_set.appliance_set_type
+            render_json_error(
+                "Unable to create appliance set with type #{appliance_set_params[:appliance_set_type]}",
```

Comment:
Line is too long. [104/80]<br>Indent the first parameter one step more than the previous line.